strapi/strapi
# fix: upgrade tool ignores node modules and dist

### What does it do?

- switches from 'glob' to 'fast-glob' to support negations
- ignore node_modules and dist paths when running codemods

### Why is it needed?

The upgrade tool was running on node_modules in plugin paths

### How to test it?

codemods should never run on files within a node_modules or dist folder
they should run everywhere else as intended

### Related issue(s)/PR(s)

DX-1599

Fixes #21165
